client.go

```diff
@@ -191,6 +191,11 @@ type Config struct {
 	// than working them. If it's specified, then Workers must also be given.
 	Queues map[string]QueueConfig
 
+	// ReindexerIndexes is a list of indexes to reindex on each run of the
+	// reindexer. If empty, defaults to only the args and metadata GIN indexes
+	// (river_job_args_index and river_job_metadata_index).
+	ReindexerIndexes []string
```

Comment:
Especially given that no one's asking for this, IMO better to leave internal for now. Just improves future flexibility and I can't think of a good reason to have a non-default.

Reply:
I think the reason would be if you want to add an additional index to the default list, like if you were doing enough throughput that `river_job_prioritized_fetching_index` was seeing a lot of bloat. But it can definitely go away for now and be re-added later when requested.